code/renderers/svelte/template/cli/js/Header.stories.js

```diff
@@ -5,24 +5,10 @@ export default {
   component: Header,
   // This component will have an automatically generated docsPage entry: https://storybook.js.org/docs/7.0/svelte/writing-docs/docs-page
   tags: ['docsPage'],
-  render: (args) => ({
-    Component: Header,
-    props: args,
-    on: {
-      login: args.onLogin,
-      logout: args.onLogout,
-      createAccount: args.onCreateAccount,
-    },
-  }),
```

Comment:
Is this related to this PR?

Reply:
Made a new PR for this https://github.com/storybookjs/storybook/pull/20007